editor/assets/effects/builtin-spine.effect

```diff
@@ -22,71 +22,60 @@ CCEffect %{
 CCProgram sprite-vs %{
   precision highp float;
   #include <cc-global>
-  #if USE_LOCAL
-    #include <cc-local>
-  #endif
 
   in vec3 a_position;
   in vec2 a_texCoord;
   in vec4 a_color;
-  in vec4 a_color2;
 
   out vec4 v_light;
-  out vec4 v_dark;
   out vec2 uv0;
 
+  #if TWO_COLORED
+    in vec4 a_color2;
+    out vec4 v_dark;
+  #endif
+
   vec4 vert () {
     vec4 pos = vec4(a_position, 1);
 
-    #if USE_LOCAL
-      pos = cc_matWorld * pos;
-    #endif
```

Comment:
Should keep USE_LOCAL define, we might use model matrix for spine, if there are no many spine copies, using model matrix would be much better than transform all vertices to world coordinates